client/rest-high-level/src/main/java/org/opensearch/client/RequestConverters.java

```diff
@@ -232,6 +235,10 @@ static Request bulk(BulkRequest bulkRequest) throws IOException {
                             metadata.field("_source", updateRequest.fetchSource());
                         }
                     }
+
+                    if (action.isRequireAlias()) {
+                        metadata.field("require_alias", action.isRequireAlias());
+                    }
```

Comment:
Does it behave differently without the `if (...)` part? Do other booleans have those conditions as well?

Reply:
Yes, 
It is creating a new index if I remove the if part.

Reply:
I meant to ask that for the `if` above, sorry.